modules/ROOT/pages/creating-a-custom-policy.adoc

```diff
@@ -0,0 +1,292 @@
+
+The JSON file must include the following fields:
+
+* Top-level @context
++
+References the vocabulary used for the extra type annotations. The aliases used for namespacing the vocabulary are a optional, but common names could be `security` or `config`.
+* Top-level title
+* $schema with a link to the JSON Schema spec (see example)
+* Root type of `object`
+* Top-level description
+* Each property must also include:
++
+** Title
++
+The title is used by API Manager UI to render the field name.
+** Description
++
+The description is used by API Manager UI to describe the field.
+** Type 
++
+Only primitive types are supported, for example `number`, `string`, `object`, `boolean`, `array`, `null`.
+
+Each property can include the following:
+
+* Validations 
++ 
+Such as patterns for strings and ranges for numbers.
+* @context with a @characteristics array
++
+Further refines the type with more annotations, such as `dataweaveExpression` or `sensitive`. These must be prefixed with namespace chosen for the vocabulary in the top-level @context.
+
+
+The following code is a example of a basic authentication JSON schema:
+
+----
+
+ {
+  "title": "Basic authentication - Simple",
+  "type": "object",
+  "description": "Enforces HTTP Basic authentication according to the details configured in the policy.",
+  "properties": {
+    "username": {
+      "title": "User Name",
+      "type": "string"
+    },
+    "password": {
+      "title": "User Password",
+      "type": "string",
+      "@context": {
+        "@characteristics": [
+          "security:sensitive"
+        ]
+      }
+    }
+  },
+  "required": [
+    "username",
+    "password"
+  ],
+  "unevaluatedProperties": false,
+  "@context": {
+    "@vocab": "anypoint://vocabulary/policy.yaml#",
+    "security": "anypoint://vocabulary/policy.yaml#"
+  },
+  "$id": "basic-authentication-simple",
+  "$schema": "https://json-schema.org/draft/2019-09/schema"
+}
+----
+
+=== Create the Definition Metadata YAML File
+
+The YAML file provides information about the policy and its requirements. The YAML file is not tied to any specific runtime or binary. 
+
+The file must have these fields configured:
+
+* Header
++
+Header must be #%Policy Definition 0.1.
+* Name
++
+User friendly name that is used for displaying the policy name in API Manager UI. Must be a string value.
+* Description
++
+Must be a string value.
+
+* Category
++
+Category of which the policy belongs. Used to group and filter policies in API Manager UI, any string value is valid. For example, security, quality of service, compliance, troubleshooting, and transformation.
+
+You can view this xref:https://github.com/mulesoft/ang-containers/blob/master/data/management/dialects/policy-definition.yaml[policy definition file] in GitHub with more information about all the supported fields. Some of the fields are auto-completed by the platform during the publication process, so not every field is read or is necessary in the file.
```

Comment:
This is not a public repository, we should not make reference to it from the documentation

Reply:
fixed